jest.config.js

```diff
@@ -60,7 +60,9 @@ module.exports = {
   // globalTeardown: undefined,
 
   // A set of global variables that need to be available in all test environments
-  // globals: {},
+  globals: {
+    AUTH_TOKEN: "test_auth_token",
```

Comment:
Need this for Jest to work, `ManualCollectionForm` has a constant in it that's normally defined on layout render but not in component-specific testing.